qa/L1_FW_iterators_separate_executor/test.h

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+# used pip packages
+pip_packages="torchvision mxnet-cu##CUDA_VERSION##"
+
+pushd ../..
+
+cd dali/test/python
+
+NUM_GPUS=$(nvidia-smi -L | wc -l)
+
+test_body() {
+    # We install it manually here because we want to limit to one version of TF
+    # depending on the CUDA version. (note: CUDA 10 is only supported in versions > 1.13)
+    if [[ "${CUDA_VERSION}" == "90" ]]; then
+        pip install tensorflow-gpu==1.12.0
+    elif [[ "${CUDA_VERSION}" == "100" ]]; then
+        pip install tensorflow-gpu==1.13.1
+    else
+        echo "Not supported CUDA version"
+        exit 1
+    fi
+
+    python test_RN50_data_pipeline.py --gpus ${NUM_GPUS} -b 256 --workers 3 --separate_queue --cpu_size 2 --gpu_size 2 --fp16 --nhwc
+
+    python test_RN50_data_fw_iterators.py --gpus ${NUM_GPUS} -b 13 --workers 3 --separate_queue --cpu_size 2 --gpu_size 2 --iters 32 --epochs 2
```

Comment:
It may be nice to test something like cpu_size=5, gpu_size=2 or 3 (different than defaults).

Reply:
Right, done